benchmark/solkz/compositional_field/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ ENDIF ()
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 SET(TARGET "solkz_compositional_field")
+SET(TARGET_DEPENDENCY "solkz")
+
 PROJECT(${TARGET})
 
+INCLUDE_DIRECTORIES("..")
+
 ADD_LIBRARY(${TARGET} SHARED solkz_compositional_field.cc)
-ASPECT_SETUP_PLUGIN(${TARGET})	
+ADD_LIBRARY(${TARGET_DEPENDENCY} SHARED ../solkz.cc)
```

Comment:
If I am correct, then `ASPECT_SOURCE_DIR` is defined by the preprocessor via `-DASPECT_SOURCE_DIR="/path/to/aspect"` (or equivalently, in some configuration header file, via `#define ASPECT_SOURCE_DIR "/path/to/aspect"`). So when you write
```
#include ASPECT_SOURCE_DIR "/benchmarks/solkz/solkz.cc"
```
then this should be expanded to
```
#include "/path/to/aspect" "/benchmarks/solkz/solkz.cc"
```
and the C++ preprocessor merges adjacent string constants into one.

At least that's what I hope will actually happen.